src/app/organizations/settings/organization-subscription.component.ts

```diff
@@ -11,6 +12,30 @@ import { PlanType } from "jslib-common/enums/planType";
 import { Organization } from "jslib-common/models/domain/organization";
 import { OrganizationSubscriptionResponse } from "jslib-common/models/response/organizationSubscriptionResponse";
 
+// Include the SVG here so that it forms part of the DOM and can use CSS color vars
+const billingHiddenSvg = `
+<svg width="216" height="231" viewBox="0 0 216 231" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Lets move the CSS into the template itself.

Reply:
Yes, I think it's just tidiest to put it in its own component, even though it feels heavyweight.

I tweaked the SVG to have a consistent `mask` property for both themes, it looked fine in Firefox but I can see not in Chrome. Will go back to design to fix it.